src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -1445,6 +1445,7 @@ class ExtConfigDBConnector(ConfigDBConnector):
     def __init__(self, ns_attrs = None):
         super(ExtConfigDBConnector, self).__init__()
         self.nosort_attrs = ns_attrs if ns_attrs is not None else {}
+        self.sub_thread_running = False
```

Comment:
>sub_thread_running [](http://example.com/codeflow?start=13&length=18)

The member is expected to be private, follow python convention, use prefix `_` or `__` to indicate the intention.

Reply:
Fixed